cfgmgr/teammgr.cpp

```diff
@@ -237,6 +237,16 @@ bool TeamMgr::checkPortIffUp(const string &port)
     return ifr.ifr_flags & IFF_UP;
 }
 
+bool TeamMgr::checkKernelPortMaster(const string &port)
+{
+    SWSS_LOG_ENTER();
+
+    string cmd = "ls /sys/class/net/" + port + "/master";
```

Comment:
It looks people recommend stat  system call for checking existence of directory/file: 

https://stackoverflow.com/questions/18100097/portable-way-to-check-if-directory-exists-windows-linux-c,

https://stackoverflow.com/questions/3828192/checking-if-a-directory-exists-in-unix-system-call